data/coalition/coalition ships.txt

```diff
@@ -1556,6 +1595,9 @@ ship "Heliarch Stalker"
 		"outfit space" 117
 		"weapon capacity" 43
 		"engine capacity" 0
+		"reverse thruster slot" 1.5
+		"steering slot" 1.5
+		"thruster slot" 1.5
```

Comment:
```suggestion
		"reverse thruster slot" 1.5
```
Has 0 engine space, seems odd to give it engine slots.

Reply:
It's entirely possible that people may develop other sorts of outfits that occupy these slots that aren't typical thrusters or steering, and perhaps that don't use engine space, either. As such, I'm going to leave them in.

That being said, since it has built in thrust and is set up explicitly as taking up the provided engine space, I'll see how many small modules that's equivalent to, and reduce the number here accordingly. I suspect 1 or 2 of each, so these will drop by 0.25 or 0.5.

Reply:
Update: the built-in thrust is about equivalent to half a small thrust module, and half a steering module. Reduced the slots for each of those categories by 0.25.